src/test/java/com/team1/dealerApp/video/movie/MovieServiceTest.java

```diff
@@ -0,0 +1,223 @@
+package com.team1.dealerApp.video.movie;
+
+
+import com.team1.dealerApp.video.Genre;
+import com.team1.dealerApp.video.VideoStatus;
+import org.apache.coyote.BadRequestException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.time.Year;
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class MovieServiceTest {
+
+    // Simula il MovieRepository
+    @Mock
+    private MovieRepository movieRepository;
+
+    // Simula il MovieMapper
+    @Mock
+    private MovieMapper movieMapper;
+
+    // Simula il MovieUpdater
+    @Mock
+    private MovieUpdater<Object> movieUpdater;
+
+    // Inietta il MovieService da testare
+    @InjectMocks
+    private MovieService movieService;
+
+    MovieDTO movieDTO;
```

Comment:
sempre meglio specificare la visibilità delle variabili, anche nei test

inoltre i nomi movieDTO, movieDTO2 ecc... non vanno bene, bisogna dare sempre nomi significativi alle variabili e non numerarle in questo modo.